oauth2client/client.py

```diff
@@ -1872,6 +1878,7 @@ def __init__(self, client_id,
         self.device_uri = device_uri
         self.token_info_uri = token_info_uri
         self.authorization_header = authorization_header
+        self.pkce = pkce
```

Comment:
Part of what @jonparrott and I are doing in our effort to improve this library is breaking consistency with bad ideas and practices of the past. Please always write your best code, not code that is consistent with the codebase to which you are contributing it.

[Default to private](https://www.youtube.com/watch?v=AFhSWQYdjy4&t=21m48s). If there's no reason for the attribute to be instance-public, leave it private.

Of course there are public and private in Python - it may be a social convention unenforced by the interpreter, but "do not access this, it may not be present in the next release of the software" is a meaningful (and powerful) message.